### DISCUSSION TITLE
Feature Request: Sponsoring contingents

### DISCUSSION THREAD

**Discussion by icepuma** [community/community#13313] [Other Feature Feedback, Questions, & Ideas]
Hey,

I'm currently contributing heavily in an OSS project. Because I run a lot of PR builds, the owner of the repo looses a lot of his `Included minutes` for Actions. It would be nice to sponsor packages of `Included minutes` for Actions or `Data transfer packages` for Packages. Something similar like `boosting` a server on `Discord`.

Best regards
Stefan

**Comment by jlord** [comment]
This is great feedback, thanks @icepuma!
